### DISCUSSION TITLE
My lax maintainership

### DISCUSSION THREAD

**Discussion by hassankhan** [serverless-heaven/serverless-webpack#1543] [Team Posts]
Hi @HyperBrain,

I'm really sorry I haven't been active on the project recently. I do appreciate all the time you put into keeping the project alive and also for making me a maintainer. However I feel I can't really be a useful maintainer of a project that I don't use in my day-to-day any more. Therefore, I was thinking of leaving the Github team.

I really hope this project finds some sort of funding or some way of sponsorship because the current situation seems unfair to you. I'm willing to ask around, but can't promise anything from my current place of employment since we've migrated from the Serverless Framework altogether 😞.

**Comment by HyperBrain** [comment]
Hi @hassankhan , I'm sorry to hear that, but I have full respect for your decision - without actually using the plugin and even the framework it is hard to help maintaining the project.
However, I'd like to say goodbye with a big "thank you". You were very engaged and helped out many people.
Maybe our paths will cross again in the future - at least I hope so 😄
